pkg/events/parse_args_helpers.go

```diff
@@ -235,10 +237,10 @@ func parseBpfAttachType(arg *trace.Argument, attachType int32) {
 	case 5:
 		attTypeName = "uretprobe"
 	default:
-		arg.Value = ""
+		arg.Value = fmt.Sprint(attachType)
```

Comment:
It would look nicer if we assigned to `attTypeName` instead of directly to `arg.Value` and remove the return, for consistency with rest of the cases.